src/main/java/io/quarkus/search/app/SearchService.java

```diff
@@ -68,6 +69,21 @@ public SearchResult<SearchHit> search(@RestQuery @DefaultValue(QuarkusVersions.L
                                         .boost(50.0f)));
                     }
                 })
+                // * Highlighters are going to use spans-with-classes so that we will have more control over styling the visual on the search results screen.
+                // * The idea is that we want to get ~5 sentences (we use a sentence boundary scanner and set the fragment size to 0)
+                //   If we decide that we want to limit that and return only parts of a sentence (if those are really long ... we limit things by setting some positive value as a fragment size.
```

Comment:
I think we do want smaller snippets.

See the UI: https://quarkus.io/guides/
We'll insert the snippets inside each card, so there's not much room, even if we replace the summary.

BTW, maybe the snippet size/count should be a parameter in the REST service? I'd have it default to count=0, meaning no highlighting at all.

Reply:
Yeah, that might be a nice option 😃 will do. I was thinking that we probably want to show the title and entire summary "no-matter-where-is-the-match" and then have a small section under with the highlights on content (small font and colors that stand out less). Let me give it a try with the UI and see 🙈

Reply:
> Yeah, that might be a nice option 😃 will do. I was thinking that we probably want to show the title and entire summary "no-matter-where-is-the-match" and then have a small section under with the highlights on content (small font and colors that stand out less).

+1, makes sense

Just FYI summaries are currently wrong in some documents, see #44 . Please ignore that problem, we'll solve it soon :)

> Let me give it a try with the UI and see 🙈

Thanks!

Reply:
Ah, and you might want to make the highlight CSS class a parameter as well... 

Maybe enable highlighting only when that parameter is set, and have the snippet size/count parameters default to something low?